crates/sui-types/src/transaction.rs

```diff
@@ -800,6 +802,25 @@ impl Command {
         }
     }
 
+    fn contains_random_object(&self) -> bool {
+        static SUI_RANDOMNESS_STATE_OBJECT_TYPE_TAG: Lazy<TypeTag> = Lazy::new(|| {
+            TypeTag::Struct(Box::from(StructTag {
+                address: SUI_FRAMEWORK_ADDRESS,
+                module: Identifier::new("random").unwrap(),
+                name: Identifier::new("Random").unwrap(),
+                type_params: vec![],
+            }))
+        });
+
```

Comment:
What are we trying to do here? This will just look for the use of random in generics?

Reply:
Am I misremembering, didn't we have a previous PR that looked for the object ID? 
Regardless, that is what we will need to do. To do this at signing, we have to:
1) Find which input (if any is the random object)
2) Look for the usage of that random object